src/utils/response.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable require-jsdoc */
+export default class Util {
+  constructor() {
+    this.statusCode = null;
+    this.type = null;
+    this.data = null;
+    this.message = null;
+  }
+
+  setSuccess(statusCode, message, data) {
```

Comment:
Please use these utility response services for your res.status

Reply:
ok